RulesEngine/Ruleset.cs

```diff
@@ -277,14 +275,26 @@ public IEnumerable<ConvertedOatRule> ByLanguage(string language)
 
         public IEnumerable<ConvertedOatRule> GetOatRules() => _oatRules;
 
-        public IEnumerable<Rule> GetAppInspectorRules() => _rules;
+        public IEnumerable<Rule> GetAppInspectorRules()
```

Comment:
This should work to replace this function and replace _rules so it works with the other functions.

```suggestion
        public IEnumerable<Rule> GetAppInspectorRules()
        {
            return _rules;
        }
        
        private IEnumerable<Rule> _rules { get =>_oatRules.Select(x => x.AppInspectorRule) }
```